html/changelogs/archive/2017-08.yml

```diff
@@ -164,7 +164,7 @@
   - balance: Lowers the chance for monkeys to become gorillas
   - rscadd: Holoparasite prompts now have a "Never For This Round" option
   Naksu:
-  - spellcheck: fixed inconsistent grammar between machines that derive from /obj/machinery/chem_dispenser
+  - spellcheck: fixed inconsistent grammar between machines that derive from /obj/machinery/atmospherics/components/unary/chem_dispenser
```

Comment:
what

Reply:
@Ghilker you botched your find/replace

Reply:
Damn i forgot to blacklist these files, my bad